release-notes/opensearch-alerting-dashboards-plugin.release-notes-1.2.0.0.md

```diff
@@ -0,0 +1,13 @@
+## Version 1.2.0.0 2021-11-04
+
+Compatible with OpenSearch Dashboards 1.2.0
+
+### Maintenance
+* Bumps version to 1.2 ([#128](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/128))
+* Cherry-pick commits from main branch to 1.x branch ([#131](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/131))
+* Added 1.2 release notes. ([#141](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/141))
+
+### Bug Fixes
+* Fixed a bug that displayed all alerts for a monitor on individual triggers' flyouts. Fixed a bug that displayed incorrect source for the condition field on the alerts flyout. Fixed a bug that displayed incorrect severity on the alerts flyout. Fixed a bug that prevented selecting query-level monitor alerts 1 by 1. Fixed bug relating to validation of popovers when defining monitor queries. ([#123](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/123))
```

Comment:
Isn't this a 1.1 change?

Reply:
Ah, you're right. I'll remove this; thank you!